apisix/cli/etcd.lua

```diff
@@ -246,9 +246,6 @@ function _M.init(env, show_output)
                 break
             end
 
-            if show_output then
```

Comment:
Why remove this?

Reply:
It would get conflict with the arguments got from `argparse`. As we discussed in #2624, I will implement the debug mode also with argument parsing in.

Reply:
Got it.